.github/workflows/test.yml

```diff
@@ -0,0 +1,38 @@
+name: Test
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v2
+      with:
+        # pytype currently doesn't work with python >= 3.9
+        python-version: 3.8
+    - name: Install tox
+      run: pip install tox
+    - name: Run style and typing checks
+      run: tox -e lint
+  test:
+    runs-on: ${{ matrix.platform }}
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9]
+        platform: [ubuntu-latest]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install tox
+      run: pip install tox
+    - name: Run the tests
+      run: tox -e py
```

Comment:
Why tox? - we don't use tox locally and the way I'm reading the config you are saying run for {python versions} so you have an environment for the right python anyway

Reply:
because it's better! I tried to explain why in the commit message https://github.com/googlefonts/picosvg/pull/145/commits/8b533eb30aea005b8524830774c7f085f9afda98

Reply:
Better than what at what? :D

Per IM I suggest documenting what concrete problems it solves in https://googlefonts.github.io/python if this is our new best practice.